automation_tools/__init__.py

```diff
@@ -1301,8 +1301,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     )
 
     # Temporary workaround to solve pulp message bus connection issue
-    # only for 6.2 and above
-    if (sat_cdn_version not in ('6.0', '6.1')):
+    # only for 6.1 and above
+    if (sat_cdn_version not in ('6.0', '6.1', '6.2')):
```

Comment:
It seems that the issues it not happening since we are skipping sat 6.0, 6.1 and 6.2. What about removing this?